File: pset1/mario/less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height; /* height var declaration*/
    do
    {
        height = get_int("Height: "); /* ask user for height and store in height*/
    }
    while (height < 1 || height > 8); /* value has to be in between 1 and 8*/

    for (int i = 1; i <= height ; i++) /* for row control*/
    {
        for (int j = height; j > i; j--) /* this handles the differences for space printing*/
        {
            printf(" ");
        }
        for (int j = 0; j < i ; j++) /* this handles for # printing at end of spaces*/
        {
            printf("#");
        }
        printf("\n"); /* print new line*/
    }
}

File: pset6/mario/more/mario.py
from cs50 import get_int

while True:  # keep repeating
    height = get_int("Height: ")  # ask for height
    if height >= 1 and height <= 8:  # get out if more than 1 or 8
        break  # get out from while

for i in range(1, height + 1):  # for row, have to +1 because it will exit at height
    for j in range(height, i, -1):  # loop for printing spaces
        print(" ", end="")
    for j in range(0, i):  # loop for printing the #
        print("#", end="")
    for gap in range(0, 2):  # loop for printing the gap
        print(" ", end="")
    for j in range(0, i):  # loop for the second #
        print("#", end="")
    print()  # print new line